src/fides/api/ctl/migrations/versions/1f61c765cd1c_merge_alembic_heads.py

```diff
@@ -10,8 +10,8 @@
 
 
 # revision identifiers, used by Alembic.
-revision = '1f61c765cd1c'
-down_revision = ('8f84fad4e00b', 'b72541d79f10')
+revision = "1f61c765cd1c"
+down_revision = ("8f84fad4e00b", "b72541d79f10")
```

Comment:
`nox -s black` formatter was complaining about this already committed code

Reply:
I have seen this happening lately with black, mypy, and pylint. Not sure why CI isn't catching them the first time (or in some cases several times) through.